libs/shared/flagd-core/src/e2e/index.ts

```diff
@@ -0,0 +1,8 @@
+export const E2E_CLIENT_NAME = 'e2e';
+
+export const IMAGE_VERSION = 'v0.5.13';
+
+export function getGherkinTestPath(file: string, modulePath = 'test-harness/gherkin/'): string {
+  // TODO: find a way to resolve this in a generic manner - currently this works, because of the file structure
```

Comment:
Is this a future TODO or something that should be resolved before merging?

Reply:
i do not think we need to handle this currently, if would be good, but this works as is, due to the layout. I am not sure if it is really worth to investigate deeper

Reply:
Could you please review the comment and resolve this thread?